sdk/storage/storage-blob-changefeed/samples/README.md

```diff
@@ -0,0 +1,83 @@
+# Getting started with samples
+
+## Install the library
+
+Copy the sample file you want to a folder of your choice and run the following command in your samples folder to install the npm package for Azure Storage Blob library.
+
+```bash
+npm install @azure/storage-blob-changefeed
+```
+
+Or, in case you have cloned the repo and want to utilize the published npm package instead of src code, run the following commands in the `samples` folder to initialize an npm project and to install the storage-blob-changefeed package.
```

Comment:
Hi Lin Jian! 👋 Here's a suggestion for this line:
```suggestion
Or, in case you have cloned the repo and want to utilize the published npm package instead of source code, run the following commands in the `samples` folder to initialize an npm project and to install the storage-blob-changefeed package.
```